addon/templates/components/entry-viewer.hbs

```diff
@@ -1,28 +1,21 @@
-<li class="entry">
-  {{#if this.isToggleable}}
-    <span class="key is-toggleable" {{action "toggleExpanded"}}>
-      <span class="key-expansion-state">
+<div class="entry-body">
+  {{~#if this.isToggleable~}}
+    <span class="key is-toggleable" {{action "toggleExpanded"}}><span class="key-expansion-state">
+      {{# if this.showIcons}}
         {{#if this.isExpanded}}
-          {{{this.expandedIcon}}}
+          <i class="fa fa-caret-down" aria-hidden="true"></i>
```

Comment:
For consumers that haven't defined style rules for `.fa.fa-caret-down` (in other words: consumers that haven't installed font-awesome), nothing renders:

![image](https://user-images.githubusercontent.com/2023/103714966-5af63500-4f8d-11eb-84ff-37bcfba5e616.png)

Reply:
(Also — if we want to hardcode expand/collapse icons that is ok, but we should also change the API for this component to not accept those properties in its displayOptions)